src/pytest/test_followhashes.py

```diff
@@ -465,3 +465,23 @@ def testExpire(env):
     conn.execute_command('EXPIRE', 'doc1', '1')
```

Comment:
@ashtul let's check right after this line that the document still exists to make sure we solved the issue we had before.